Dockerfile

```diff
@@ -1,15 +1,28 @@
-FROM ipython/scipystack
+FROM ubuntu:14.04
 
-ADD . /srv/runner/
-WORKDIR /srv/runner/
-
-RUN pip2 install .
+MAINTAINER Kyle Kelley <rgbkrk@gmail.com>
 
+RUN apt-get update && apt-get install -y wget bzip2
 RUN useradd -m -s /bin/bash rho
 
+RUN echo 'export PATH=/home/rho/anaconda:$PATH' > /etc/profile.d/conda.sh
```

Comment:
I like less layers when I'm working with these, but not when I'm reading them or wishing the steps were cached individually. It doesn't take that much extra space on disk, just more time for downloading and more images listed when doing a `docker images -a`.